model-inference/dockerfiles/base_model/requirements.txt

```diff
@@ -15,3 +15,5 @@ filelock==3.4.2
 httpx
 setuptools>=65.5.1 # not directly required, pinned by Snyk to avoid a vulnerability
 nltk
+bertviz
+ipython
```

Comment:
is ipython really a requirement this service needs?

Reply:
Hi Tim, yes ipython is needed for bertviz. Without ipython we will run into an error.

Reply:
I believe ipython is only needed for running it on a Jupyter Notebook/Google Colab

Reply:
I checked the bertviz code, unfortunately they import ipython functionalities at the top, no matter if you use them or not (eg [here](https://github.com/jessevig/bertviz/blob/04755efcf8e87980a19e31a5c19a1035c52e3978/bertviz/model_view.py#L7)). So I guess it is a requirement. (Though if you want to contribute to open source, this might be a good thing to work on:))

Reply:
Hi guys, sorry I couldn't reply to you on time. But still many thanks for the merging!